extensions/wikia/UserRenameTool/RenameUserProcess.class.php

```diff
@@ -699,14 +653,23 @@ public function updateLocal() {
 			// Determine all namespaces which need processing
 			$allowedNamespaces = array( NS_USER, NS_USER_TALK );
 			// Blogs extension
-			if ( defined( 'NS_BLOG_ARTICLE' ) )
+			if ( defined( 'NS_BLOG_ARTICLE' ) ) {
 				$allowedNamespaces = array_merge( $allowedNamespaces, array( NS_BLOG_ARTICLE, NS_BLOG_ARTICLE_TALK ) );
+			}
 			// NY User profile
```

Comment:
Below I did three things.  First, I added brackets per coding conventions.  Second I got rid of ```NS_USER_PROFILE``` from the array because I don't see it defined anywhere.  Lastly I reformatted the long line to several lines.